tests/test_tracebacks.py

```diff
@@ -531,6 +531,40 @@ def bar(n):
     ]
 
 
+@pytest.mark.parametrize(
+    ("kwargs", "local_variable"),
+    [
+        (
+            {"locals_max_string": None},
+            "x" * (tracebacks.LOCALS_MAX_STRING + 10),
+        ),
+        (
+            {"locals_max_length": None},
+            list(range(tracebacks.LOCALS_MAX_LENGTH + 10)),
+        ),
+    ],
+)
+def test_exception_dict_transformer_locals_max_accept_none_argument(
+    kwargs, local_variable
+):
```

Comment:
Sorry for the late response! The CI is failing because your test doesn't have a docstring in the form of https://jml.io/test-docstrings/

See also: https://hynek.me/articles/document-your-tests/

Reply:
Alright, got it, thanks. I tried to add a suitable docstring, hope that suffices.